### PR TITLE
Ios sdk changes

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: sdks/paths.mk [hunk 1]
```diff
@@ -42,10 +42,8 @@ ifndef DISABLE_IOS
 
 CheckXcodeDir=$(or $(and $(wildcard $(1))),$(warning Could not find Xcode in "$(1)"))
 
-ifneq ($(call CheckXcodeDir,/Applications/Xcode.app/Contents/Developer),)
-XCODE_DIR=/Applications/$(XCODE_VERSION).app/Contents/Developer
-else
-$(error Could not find Xcode)
+ifeq ($(wildcard $(XCODE_DIR)),)
```
Comment (luhenry) [inline sdks/paths.mk]: Could we just keep the call to `CheckXcodeDir` here or does it breal something?

Comment (vargaz) [reply sdks/paths.mk]: What does that call actually do here ?

Comment (vargaz) [reply sdks/paths.mk]: So I think we should error out if the xcode path doesn't exist, thats why the call was removed.

Comment (luhenry) [reply sdks/paths.mk]: It keeps it consistent with the Android NDK and SDK check. We do fail in the previous implementation, right after printing the warning.

**Comment by marek-safar** [comment]
@monojenkins backport master

**Pull request merged**